### ISSUE TITLE
PC wakes from sleep, TB is in safe mode with a message to use an option "to fix it" 

### ISSUE THREAD

**Issue by yereverluvinunclebert** [twinbasic/twinbasic#1951]
Whenever my 'puter wakes up from a asleep I find that TB is in some sort of safe mode with a message to use an option "to fix it" that I have not yet figured out. I typically close the TB IDE and restart. 

Workaround from Wayne Phillips : To get out of safe mode without restarting the IDE, you can change the target architecture dropdown back to 'Win32' or 'Win64' as appropriate (in the IDE toolbar).

Beta 622 on Windows 10 22H2

**Comment by fafalone** [comment]
Yeah I'd been noticing this on my tablet but just chalked it up to one of the many touchscreen problems, since my desktop never sleeps.

**Comment by yereverluvinunclebert** [comment]
Wayne never sleeps.

**Comment by yereverluvinunclebert** [comment]
Update. This used to happen every time the system was placed into a sleep mode. It no longer does. No change to the system nor to TwinBasic. No change as far as I am aware. It just stopped occurring. 

Edition	Windows 10 Pro
Version	22H2
Installed on	‎05/‎08/‎2023
OS build	19045.4780
Experience	Windows Feature Experience Pack 1000.19060.1000.0

Windiows update switched OFF

**Comment by yereverluvinunclebert** [comment]
It is doing it again :)

**Comment by yereverluvinunclebert** [comment]
It is possibly due to this error:

IDE internal error: Uncaught TypeError: Cannot set properties of undefined (setting 'openEditorNode'), 1, 408912, TypeError: Cannot set properties of undefined (setting 'openEditorNode')

**Comment by yereverluvinunclebert** [comment]
In addition, this error keeps popping up in the debug console:
```
IDE internal error: Uncaught TypeError: Cannot read properties of undefined (reading 'toolTip'), 1, 697160, TypeError: Cannot read properties of undefined (reading 'toolTip')
```

It might be related, or not.

It references a .tooltip property that is used by a richClient widget so it should not be being generated.

**Comment by WaynePhillipsEA** [comment]
This should now be fixed as of BETA 846 (tested on Win 11 only).  I will await further confirmations before closing the issue.

**Comment by yereverluvinunclebert** [comment]
I have not encountered this issue over the last few wake from sleeps. Feel free to accept the closure.

**Comment by fafalone** [comment]
Oops yeah forgot to report back, it's been ok on my tablet. There's a few compiler restart messages in the console, but it's not in safe mode or otherwise not working. All good 👍
